### PR TITLE
[DataX] Refactor doriswriter

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

1. Use `read_json_by_line` to load data
2. Reorganized the parameters required by the plug-in.
3. Reorganized the DataX compilation environment.
4. Optimized some code logic
5. Use FE http server as the target host of stream load

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — COMMENTED** [review]
OK

**Comment by morningman** [comment]
@huzk8 Please review if you have time

File: extension/DataX/doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/Key.java [hunk 1]
```diff
@@ -92,32 +108,43 @@ public String getPassword() {
...
+        }
+        if (urlList == null || urlList.isEmpty()) {
+            throw DataXException.asDataXException(DBUtilErrorCode.CONF_ERROR, "Either beLoadUrl or feLoadUrl must be set");
+        }
+
         for (final String host : urlList) {
             if (host.split(":").length < 2) {
-                throw DataXException.asDataXException(DBUtilErrorCode.CONF_ERROR, "loadUrl的格式不正确，请输入 `be_ip:be_http_ip;be_ip:be_http_ip`。");
+                throw DataXException.asDataXException(DBUtilErrorCode.CONF_ERROR,
+                        "Invalid load url format. Should be host:port");
```
Comment (pengxiangyu) [inline extension/DataX/doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/Key.java:139]: 这里还是写：be_ip:be_http_ip;be_ip:be_http_ip吧

Comment (morningman) [reply extension/DataX/doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/Key.java]: done

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**